### ISSUE TITLE
Can't import `library` module from `finetune/make_captions.py` using docker

### ISSUE THREAD

**Issue by ericwu13** [bmaltais/kohya_ss#1007]
I am using the docker env provided from the repo and tried to do BLIP Captioning. I got the errors below

```
2023-06-18 08:28:14.304207: E tensorflow/stream_executor/cuda/cuda_blas.cc:2981] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
╭─────────────────────────────── Traceback (most recent call last) ────────────────────────────────╮
│ /app/finetune/make_captions.py:20 in <module>                                                    │
│                                                                                                  │
│    17                                                                                            │
│    18 sys.path.append('/app/library')                                                            │
│    19                                                                                            │
│ ❱  20 import library.train_util as train_util                                                    │
│    21                                                                                            │
│    22 DEVICE = torch.device("cuda" if torch.cuda.is_available() else "cpu")                      │
│    23                                                                                            │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
ModuleNotFoundError: No module named 'library'
```

I resolved this error by adding in the `finetune/make_captions.py`
```
import sys
sys.path.insert(0, '/app')
```

I am sure there might be other files that have this include errors. I am not sure if this is a clean way to address it, can someone help take a look and see if there is a cleaner way?

**Comment by joshuashaffer** [comment]
Set `$PYTHONPATH` in `docker-compose.yml` ...

```diff
diff --git a/docker-compose.yaml b/docker-compose.yaml
index c24588f..07a2b85 100644
--- a/docker-compose.yaml
+++ b/docker-compose.yaml
@@ -6,17 +6,19 @@ services:
     build:
       context: .
     ports:
      - 127.0.0.1:7860:7860
     tty: true
     ipc: host
     environment:
       CLI_ARGS: ""
       SAFETENSORS_FAST_GPU: 1
       DISPLAY: $DISPLAY
+      PYTHONPATH: /app
     tmpfs:
       - /tmp      
     volumes:
       - ./dataset:/dataset
       - ./.cache/user:/home/appuser/.cache
       - ./.cache/triton:/home/appuser/.triton    
       - ./.cache/config:/app/appuser/.config

```

**Comment by GuillaumeCleme** [comment]
Can confirm that this issue is fixed as of [e084bc55](https://github.com/bmaltais/kohya_ss/commit/e084bc55395ba85b6aca4fdd897a7c474f3632ae)
